iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -211,6 +211,46 @@ static LogicalResult setRootConfig(FuncOp entryPointFn,
       IREE::HAL::DispatchLoweringPassPipeline::CPUVectorization);
 }
 
+/// Sets the lowering configuration for dispatch region with root op being a
+/// `ConvolutionOpInterface`.
+static LogicalResult setRootConfig(FuncOp entryPointFn,
+                                   linalg::ConvolutionOpInterface convOp) {
```

Comment:
I actually want to avoid such op-specific control. That is very hard to maintain. So trying to switch to use general interface by default, and specialize if we need.